clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -515,7 +515,7 @@
         "package": "vsce package"
     },
     "dependencies": {
-        "@zowe/zowe-explorer-api": "^2.3.0",
+        "@zowe/zowe-explorer-api": "^2.4.0",
```

Comment:
```suggestion
        "@zowe/zowe-explorer-api": "2.4.0",
```

Reply:
Let's have the exact version here to avoid surprises with implicit version updates.